2022-02-03 17:32:13,372 INFO    MainThread:457908 [wandb_setup.py:_flush():71] setting env: {}
2022-02-03 17:32:13,372 INFO    MainThread:457908 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-03 17:32:13,372 INFO    MainThread:457908 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-03 17:32:13,372 INFO    MainThread:457908 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\aar245\Desktop\privacy_preserving_nn\src_2\wandb\run-20220203_173213-j8fzihhu\logs\debug.log
2022-02-03 17:32:13,373 INFO    MainThread:457908 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\aar245\Desktop\privacy_preserving_nn\src_2\wandb\run-20220203_173213-j8fzihhu\logs\debug-internal.log
2022-02-03 17:32:13,399 INFO    MainThread:457908 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000020BE005CDF0>
2022-02-03 17:32:13,399 INFO    MainThread:457908 [wandb_init.py:init():404] calling init triggers
2022-02-03 17:32:13,400 INFO    MainThread:457908 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'appliance': 'refrigerator1', 'epochs': 1500, 'batch_size': 150, 'learning_rate': 0.0003318089969908341, 'in_channels': 1, 'out_channels': 16, 'kernel_size': 7, 'hidden_size_1': 64, 'hidden_size_2': 85, 'fc1': 211, 'fc2': 109, 'weight_decay': 0.08044407443013193, 'window_size': 136}
2022-02-03 17:32:13,400 INFO    MainThread:457908 [wandb_init.py:init():460] starting backend
2022-02-03 17:32:13,400 INFO    MainThread:457908 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-02-03 17:32:13,406 INFO    MainThread:457908 [backend.py:ensure_launched():216] starting backend process...
2022-02-03 17:32:13,534 INFO    MainThread:457908 [backend.py:ensure_launched():221] started backend process with pid: 458952
2022-02-03 17:32:13,535 INFO    MainThread:457908 [wandb_init.py:init():469] backend started and connected
2022-02-03 17:32:13,559 INFO    MainThread:457908 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-03 17:32:13,563 INFO    MainThread:457908 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-03 17:32:13,563 INFO    MainThread:457908 [wandb_init.py:init():533] updated telemetry
2022-02-03 17:32:13,564 INFO    MainThread:457908 [wandb_init.py:init():563] communicating current version
2022-02-03 17:32:15,531 INFO    MainThread:457908 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-03 17:32:15,532 INFO    MainThread:457908 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-03 17:32:15,734 INFO    MainThread:457908 [wandb_init.py:init():606] starting run threads in backend
2022-02-03 17:32:20,748 INFO    MainThread:457908 [wandb_run.py:_console_start():1810] atexit reg
2022-02-03 17:32:20,750 INFO    MainThread:457908 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-03 17:32:20,750 INFO    MainThread:457908 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-03 17:32:20,753 INFO    MainThread:457908 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-03 17:32:20,754 INFO    MainThread:457908 [wandb_init.py:init():633] run started, returning control to user process
2022-02-03 17:32:23,813 INFO    MainThread:457908 [wandb_watch.py:watch():43] Watching
2022-02-04 05:01:03,437 INFO    MainThread:457908 [wandb_run.py:finish():1457] finishing run nilm/jan27_FL_trials/j8fzihhu
2022-02-04 05:01:03,445 INFO    MainThread:457908 [jupyter.py:save_history():458] saving 14 cells to _session_history.ipynb
2022-02-04 05:01:03,452 INFO    MainThread:457908 [wandb_run.py:_config_callback():956] config_cb ('_wandb', 'session_history') code\_session_history.ipynb None
2022-02-04 05:01:03,473 INFO    MainThread:457908 [jupyter.py:save_ipynb():377] looking for notebook: Desktop/privacy_preserving_nn/src_2/main_fed.ipynb
2022-02-04 05:01:03,473 INFO    MainThread:457908 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-04 05:01:03,473 INFO    MainThread:457908 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-04 05:01:03,474 INFO    MainThread:457908 [wandb_run.py:_restore():1752] restore
2022-02-04 05:01:04,332 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-02-04 05:01:04,535 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-02-04 05:01:04,705 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-02-04 05:01:04,998 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 1403713
}

2022-02-04 05:01:05,102 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 1403713
}

2022-02-04 05:01:05,205 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 196492
  total_bytes: 1403713
}

2022-02-04 05:01:05,309 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1403713
  total_bytes: 1403713
}

2022-02-04 05:01:05,413 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1403713
  total_bytes: 1403713
}

2022-02-04 05:01:05,517 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1403713
  total_bytes: 1403713
}

2022-02-04 05:01:05,620 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1403713
  total_bytes: 1403713
}

2022-02-04 05:01:05,722 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1403713
  total_bytes: 1403713
}

2022-02-04 05:01:06,189 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1403713
  total_bytes: 1403713
}

2022-02-04 05:01:06,510 INFO    MainThread:457908 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1403713
  total_bytes: 1403713
}
local_info {
}

2022-02-04 05:01:07,653 INFO    MainThread:457908 [wandb_run.py:_append_history():2130] rendering history
2022-02-04 05:01:07,655 INFO    MainThread:457908 [wandb_run.py:_append_summary():2085] rendering summary
2022-02-04 05:01:07,656 INFO    MainThread:457908 [wandb_run.py:_append_files():2180] logging synced files
